ui-kit.code-workspace

```diff
@@ -0,0 +1,123 @@
+{
+  "folders": [
+    {
+      "name": "✨ ui-kit",
+      "path": "./"
+    },
+    {
+      "name": "📦 Auth",
```

Comment:
Does this order reflect in VSCode UI as well? If yes then can we order by importance / frequency-of-editing .. e.g. headless, atomic, followed by integration packages e.g. atomic-react, followed by samples etc